### DISCUSSION TITLE
Icon Request: icon-case-sensitive, icon-whole-word, icon-regex

### DISCUSSION THREAD

**Discussion by mdo77** [FortAwesome/Font-Awesome#21152] [Icon Requests]
I'm implementing a search dialog in my application and it would be nice to have icons similar to what's in the visual studio '12 search dialog.  See attached image.  The three icons are for case sensitive, whole word and regular expression searches

![image](https://f.cloud.github.com/assets/4448997/513471/cdab1e82-be3e-11e2-9fa0-33494d2114c9.png)

**Comment by cloose** [comment]
:+1:

**Comment by adamjimenez** [comment]
:+1:

**Comment by valtlai** [comment]
:+1:

**Comment by mchaput** [comment]
I've created SVGs for these icons in this repo: https://github.com/mchaput/icons, if it helps.

**Comment by adamjimenez** [comment]
who do I need to blow to get these included?

**Comment by abnerlee** [comment]
:+1:

**Comment by mauriciokj** [comment]
+1

**Comment by MPLew-is** [comment]
+1

**Comment by mtrenkmann** [comment]
+1 for fa-case-sensitive

**Comment by Gitoffthelawn** [comment]
+1

**Comment by BramVanroy** [comment]
+1 for the case-sensitive one!

**Comment by diggyk** [comment]
+1 for regex

**Comment by simneol** [comment]
+1

**Comment by MaralMosayebi** [comment]
+1

**Comment by canpan14** [comment]
+1

**Comment by gillez** [comment]
+1 (at the moment for regex, but they would all be useful).

**Comment by santhikumar43** [comment]
+1

**Comment by vince83110** [comment]
+1 for all of them, can't believe they are not available...

**Comment by tagliala** [comment]
Font-case is available in FA Pro >= 5.9.0: https://fontawesome.com/icons/font-case?s=solid&f=classic

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂
